### DISCUSSION TITLE
解决火狐（版本大于107）兼容性问题 | 八阿哥的博客

### DISCUSSION THREAD

**Discussion by giscus** [yuuuuuyu/bugs#5] [Announcements]
# 解决火狐（版本大于107）兼容性问题 | 八阿哥的博客

bug大集合

http://localhost:5173/bugs/posts/2024/08/06%E8%A7%A3%E5%86%B3Firefox%E6%B5%8F%E8%A7%88%E5%99%A8popover%E7%BB%84%E4%BB%B6%E5%85%BC%E5%AE%B9%E6%80%A7%E9%97%AE%E9%A2%98.html

**Comment by yuuuuuyu** [comment]
1
